src/hooks/useKeyboardState/index.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+
+import { KeyboardEvents } from "../../bindings";
+import { KeyboardController } from "../../module";
+
+// TODO: expose isVisible, isTransitioning (isHiding/isShowing) props?
+export const useKeyboardState = () => {
+  const [state, setState] = useState(KeyboardController.state());
+
+  useEffect(() => {
+    const didShowSubscription = KeyboardEvents.addListener(
+      "keyboardDidShow",
+      (e) => {
+        setState(e);
+      },
```

Comment:
you don't need to create a new function for listener

```suggestion
     setState
```